README.md

```diff
@@ -1 +1,43 @@
-# reservation
\ No newline at end of file
+# Reservation Module
```

Comment:
I'd recommend including your endpoint to get static assets (e.g., the users initial entry into the application, how users of your microservice will request the bundle.js, etc.)